src/components/Comments/Comment.js

```diff
@@ -44,6 +45,7 @@ const Comment = ({
   showLeftConnector = true,
   disableOnTextPress = false,
   onLongPress,
+  feedId,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `feedId` prop has been added to the component. Ensure that all instances where this component is used are updated to pass this prop.



<!-- This is an auto-generated comment by OSS CodeRabbit -->